src/hub.js

```diff
@@ -352,14 +370,31 @@ const onReady = async () => {
   const initialEnvironmentEl = document.createElement("a-entity");
   initialEnvironmentEl.addEventListener("bundleloaded", () => {
     remountUI({ initialEnvironmentLoaded: true });
-    // Wait a tick plus some margin so that the environments actually render.
-    setTimeout(() => scene.renderer.animate(null), 100);
+    // We never want to stop the render loop when were running in "bot" mode.
+    if (!qsTruthy("bot")) {
+      // Stop rendering while the UI is up. We restart the render loop in enterScene.
+      // Wait a tick plus some margin so that the environments actually render.
+      setTimeout(() => scene.renderer.animate(null), 100);
+    }
   });
   environmentRoot.appendChild(initialEnvironmentEl);
 
+  const setRoom = (janusRoomId, hubName) => {
+    if (qsTruthy("bot")) {
```

Comment:
I'd either flip the conditions here to put the non-bot case first, or leave it as is and just return early (perhaps extracting the bot scene entry mechanics into a new function) and drop the `else`. (the reason being I like to have exceptional conditions (like bot mode) come after the main code path or be early exit'ed via a guard clause, but just style thing probably)

Reply:
Makes sense. I'll flip the conditional.